### ISSUE TITLE
Column doesn't exist for error message on NAs in predictors

### ISSUE THREAD

**Issue by sashahafner** [sashahafner/ALFAM2#46]

**Comment by sashahafner** [comment]
Fixed in 0bfcf09df9a6128295efb11965073a7b2bb0f311 (Rcpp-dev). Was probably caused by changes made in Rcc-dev work (I removed unused cols).

**Comment by sashahafner** [comment]
Actually is OK. 
I did not have latest Rcpp-dev version installed (v2.3).
Once I have that, I see this:

```
> dpred1 <- alfam2(as.data.frame(bdat), pars = ALFAM2pars01, app.name = 'tan.app
', time.name = 'cta', group = 'pmid', prep = TRUE)
User-supplied parameters are being used.
Error!
Missing values in predictors:
app.mthd.os    app.rate      man.dm    air.temp     wind.2m      man.ph
          0           0           0           0           0         342
  rain.rate    rain.cum
          0           0
Error in alfam2(as.data.frame(bdat), pars = ALFAM2pars01, app.name = "tan.app",
 :
  NA values in primary parameters. Look for missing values in predictor variable
s (in dat) and double-check parameters agaist dat column names

```

This is from some early work on eGylle2/Kamp stuff.
